src/v3/src/img/yubikeyDemo.svg

```diff
@@ -0,0 +1,19 @@
+<svg width="91" height="85" viewBox="0 0 91 85" fill="none" xmlns="http://www.w3.org/2000/svg" aria-labelledby="yubikeyDemo">
+<title id="yubikeyDemo">yubikey</title>
```

Comment:
Could you stick a FIXME/TODO note here that the `<title>` needs to be i18n/l10n-friendly?